tests/compose

```diff
@@ -1,9 +1,31 @@
 #!/bin/bash
-
-set -xeuo pipefail
+set -euo pipefail
 
 dn=$(cd $(dirname $0) && pwd)
 
+LOG=${LOG:-compose.log}
+date > ${LOG}
+
+colour_print() {
+  colour=$1; shift
+  [ ! -t 1 ] || echo -en "\e[${colour}m"
+  echo -n "$@"
+  [ ! -t 1 ] || echo -en "\e[0m"
+  echo
+}
+
+pass_print() {
+  colour_print 32 "$@" # green
+}
+
+fail_print() {
+  colour_print 31 "$@" # red
+}
+
+skip_print() {
+  colour_print 34 "$@" # blue
+}
+
```

Comment:
This stuff is more or less copied from `vmcheck/test.sh`. We can probably unify them somewhere.